### ISSUE TITLE
Can Rubberduck fix the phantom break point issue in VBA

### ISSUE THREAD

**Issue by Stevefb** [rubberduck-vba/Rubberduck#4163]
Sometimes VBA code execution stops spontaneously and for no apparent reason, it is as if debugging break points are set but there are none. This [thread explains the issue and the highest voted answer seems to provide a robust solution](https://stackoverflow.com/questions/2154699/excel-vba-app-stops-spontaneously-with-message-code-execution-has-been-halted): 

> Press "Debug" button in the popup.
> Press Ctrl+Pause|Break twice.
> Hit the play button to continue.

**Comment by ThunderFrame** [comment]
The approach listed in the linked answer would seem to work, but I suspect that the problem is because the p-code has become corrupted, and needs to be cleaned, as per Charles William's [answer](https://stackoverflow.com/a/2155022/5757159). IIRC, RD removes and re-imports most modules as part of the parsing process, effectively the same thing as the Code Cleaner approach, but for any VBA host, not just Excel. That is, Rubberduck should already be pre-empting this problem. The re-import forces VBA to recompile the code for the current version and bitness of VBA.

If I'm mistaken about the removal/re-import steps, there is an open issue for making the cleaning process more explicit: [Rubberduck should add a Decompile/Clean Project command](https://github.com/rubberduck-vba/Rubberduck/issues/1752)

Can you confirm that you're having code stop spontaneously, after parsing with Rubberduck? Or is this just a feature request for a feature that we've inadvertantly implemented?

**Comment by Stevefb** [comment]
@ThunderFrame thanks for your thoughts. Yes I can confirm that I refreshed Rubberduck in a deliberate effort to resolve the phantom breaks but without success.  

FWIW I understand what you mean WRT re-importing files; ages ago I noticed that my recent changes to code 'evaporated' if, for example, I attempted to run code fixes without first refreshing Rubberduck... I attributed that to Rubberduck re-importing an older version of the file.  

Also, I'm at a loss as to how the phantom breaks appeared - there were so many of them and on a relatively new project (days old) that only I had edited...
